app/lib/features/activities/widgets/notification_card.dart

```diff
@@ -125,12 +129,16 @@ class NotificationCard extends ConsumerWidget {
         onTap: () {
           switch (brief.route) {
             case Routes.chatroom:
-              context.pushNamed(
-                Routes.chatroom.name,
-                pathParameters: {
-                  'roomId': roomId,
-                }, // FIXME: fails at the moment
-              );
+              ref.read(roomIdProvider.notifier).update((state) => roomId);
+              isDesktop
+                  ? context.goNamed(
+                      Routes.chatroom.name,
+                      pathParameters: {'roomId': roomId},
+                    )
+                  : context.pushNamed(
```

Comment:
Yeah there could be function which sorts of handle the navigation stack on what the platform is running. But I think in the long run, this would also not be beneficial. I am of the idea that we should keep two go routers with separate navigation logic and loading up one from the root during app launch, based on what platform is of the context. This would simplify the core navigation and easier to apply changes without making them (interdependent) and also not introducing more problems underway.

Reply:
as pointed out below, I think anything that goes for a `isDesktop`-Style check will ultimately fail us as it doesn't take into account the complexity that is responsive layouts ... like tables in landscape format ... not a desktop, not a mobile either ...